Userland/Demos/VirGLDemo/CommandBufferBuilder.cpp

```diff
@@ -235,9 +237,9 @@ void CommandBufferBuilder::append_set_constant_buffer(Vector<float> const& const
     }
 }
 
-void CommandBufferBuilder::append_create_shader(ObjectHandle handle, Gallium::ShaderType shader_type, const char* shader_data)
+void CommandBufferBuilder::append_create_shader(ObjectHandle handle, Gallium::ShaderType shader_type, StringView shader_data)
 {
-    size_t shader_len = strlen(shader_data) + 1; // Need to remember to copy null terminator as well if needed
+    size_t shader_len = shader_data.length() + 1; // Need to remember to copy null terminator as well if needed
```

Comment:
I would like an explanation of why this change is also done in the commit description, as I had to figure it out myself (and it's completely acceptable).